angular/projects/spark-angular/src/lib/components/sprk-masthead/directives/sprk-masthead-link/sprk-masthead-link.directive.ts

```diff
@@ -26,7 +26,8 @@ export class SprkMastheadLinkDirective implements OnInit {
   idString: string;
 
   /**
-   * Will cause the appropriate variant type to render.
+   * If `navBar` is supplied then the link style
```

Comment:
It's redundant to say "if navBar is supplied" because it's already about navBar Input. We should focus on what this input does specifically.

`Decides the link style of the nav bar links.`

Reply:
Do we need to say "if omitted...." aren't defaults already a given? I imagine people go here to learn how to change the default.

Reply:
@RVMendoza hmm, this is the masthead link directive and not the nav bar directive. So its about masthead links and not nav bar links in the masthead. The nav bar link is one type of link that could exist in the masthead.